src/control/server/storage/scm/ipmctl.go

```diff
@@ -194,8 +194,10 @@ func (cr *cmdRunner) processActionableState(req storage.ScmPrepareRequest, state
 	switch state {
 	case storage.ScmNoRegions:
 		// No regions exist, create interleaved AppDirect PMem regions.
+		// PMem mode should be changed for all sockets as changing just for a single socket
+		// can be determined as an incorrect configuration on some platforms.
```

Comment:
I'm curious to know more about this. Is the issue that we're only setting goals for one socket, or that the two sockets are potentially different?

Reply:
https://daosio.atlassian.net/browse/SRE-483 was the issue that appeared (details in the comment), where the node wouldn't boot after resetting only one of the socket's PMem. Wolf-157, CLX and the command was `ipmctl create -f -socket 1 -goal MemoryMode=100` which should have resulted in socket 0 in AppDirect mode and socket 1 in Memory mode. The commands may not be exactly that listed above as I didn't record them but something very similar.

Reply:
Interesting! Thanks for the explanation.